test/unittests/instantiate_test.cpp

```diff
@@ -419,7 +426,8 @@ TEST(instantiate, memory_single_unspecified_maximum)
     auto instance = instantiate(module);
 
     ASSERT_EQ(instance->memory->size(), PageSize);
-    EXPECT_EQ(instance->memory_max_pages * PageSize, 256 * 1024 * 1024);
+    EXPECT_EQ(instance->memory_limits.min, 1);
+    EXPECT_FALSE(instance->memory_limits.max.has_value());
```

Comment:
This is a bit misleading though, because we do have a hard limit, albeit I believe this is more in line with the standard, that that hard limit is enforced only during `memory.grow`.